packages/forma-36-react-components/src/components/IconButton/IconButton.tsx

```diff
@@ -59,7 +85,7 @@ export const IconButton = ({
   testId = 'cf-ui-icon-button',
   withDropdown = false,
   ...otherProps
-}: IconButtonProps) => {
+}: IconButtonProps & IconButtonAnchorProps) => {
```

Comment:
@denkristoffer thx for the feedback! 🙂 is this how I supposed to be doing that?

Reply:
I think the other change should work too, but if we're removing IconButton in v4 anyway it might not be worth messing with.

Reply:
@denkristoffer sure, added. I would keep maybe both changes, so we are sure that target does not appear without href.